index.html

```diff
@@ -0,0 +1,198 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="css/reset.css">
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style.css" >
+  </head>
+  <body>
+    <!--Header Section (Toolbar)-->
+    <header class="header backgrnd-clr">
+      <nav class="navbar">
+        <a href="index.html"><img src="images/icons/Welcome.png" alt="logo" /></a>
+        <img src="images/icons/Union.png" alt="hamburger-menu" />
+      </nav>
+    </header>
+    <!--Main content-->
+    <main class="main-content">
+      <!--Headline Section-->
+      <section class="head-line backgrnd-clr">
+        <!--Greeting text-->
+        <h2 class="greeting greeting-text headline-box">
+          Hey there. I’m <br />
+          Mariane
+          <span class="text-color">I’m a software developer</span>
+        </h2>
+        <!--Supporting text-->
+        <p class="supportig-text headline-box">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <!--social media links-->
+        <section  class="social-media headline-box">
+          <a href="https://twitter.com/Elad59380989" class="social-links"
+            ><img src="images/icons/twitter.svg" alt="twitter"
+          /></a>
+          <a
+            href="https://www.linkedin.com/in/tumbu-elad-896ab2183/"
+            class="social-links"
+            ><img src="images/icons/Linkedin.svg" alt="linkedin"
+          /></a>
+          <a href="#" class="social-links"
+            ><img src="images/icons/medium.svg" alt="medium"
+          /></a>
+          <a href="https://github.com/elad237" class="social-links"
+            ><img src="images/icons/github.svg" alt="github"
+          /></a>
+          <a href="#" class="social-links"
+            ><img src="images/icons/angellist.svg" alt="angellist"
+          /></a>
+        </section >
+      </section>
+
+      <section class="portfolio-wrapper" id="portfolio">
+        <h2 class="portfolio-header">My Recent Works</h2>
+        <section class="portfolio-project">
+            <figure class="project-item">
```

Comment:
As represented in Figma, please add:
- The green line under 'My Recent Works'.
- More spacing between the header and the image.
- The circle containing the down arrow. 😉

Figma design:
![image](https://user-images.githubusercontent.com/42799579/193402809-b7bb7406-da4e-4e1b-aa12-fdfb0e90f0c5.png)

Your design:
![image](https://user-images.githubusercontent.com/42799579/193402813-bc1a9ee9-f779-492a-b7ef-8ed3a807289f.png)